lib/BadgeOverlaySupport/BadgeOverlaySupport.less

```diff
@@ -0,0 +1,25 @@
+.moon-badge-overlay-support {
+	.moon-image-client {
```

Comment:
Hi @ryanjduffy . The other way i can think of accessing the client is accessing it via component and adding class (image.$.client). But it looks like breaking the encapsulation. 'moon-image' class can be removed and can be achieved same effect via the code, but 'moon-image-client' , need some more analysis.
Thankyou.